Earthfile

```diff
@@ -768,7 +768,9 @@ datasource-iso:
 trivy:
     ARG TRIVY_VERSION
     FROM aquasec/trivy:$TRIVY_VERSION
+    RUN /usr/local/bin/trivy fs --download-db-only
     SAVE ARTIFACT /contrib contrib
+    SAVE ARTIFACT /root cache
```

Comment:
maybe you meant a specific directory in `/root` here?

Reply:
well, indeed its `/root/.cache` the one with the actua DB but I thought it would be helpful to capture it all just in case there is more stuff needed in there?

Reply:
but then you don't want to copy it at `/root/.cache` below, no? I mean, you would be copying `/root` in `/root/.cache` which would result in a directory structure like `/root/.cache/.cache`.

Reply:
ummm...good point lol I did nothing!